frontend/src/pages/team/dialogs/ConfirmTeamDeleteDialog.vue

```diff
@@ -24,7 +24,7 @@
                                         Are you sure you want to delete this team? Once deleted, there is no going back.
                                     </p>
                                     <p class="text-sm text-gray-500">
-                                        Enter the team name '<code>{{team.name}}</code>' to continue.
+                                        Enter the team name '<code class="block">{{team.name}}</code>' to continue.
```

Comment:
I think we can also remove the `'` either side of the `<code>` block as it looks a little strange with them still in there:

<img width="366" alt="Screenshot 2022-04-06 at 12 11 24" src="https://user-images.githubusercontent.com/99246719/161962448-87ccc636-eebb-401a-8d7a-931705b56df1.png">

Reply:
resolved

Reply:
Thanks! We had some other instances of this styling needing to be applied on a couple of other dialogs, mentioned back in the comment thread on #424 - if you can get those in for this PR then I think we'll be good to go after that